jkamz/Questioner
# #163069134 create directory structure

### What does this PR do?
This PR creates the directory structure

### Description of the tasks to be completed
The tasks for this PR include:

- Create folders and include init files

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ch-folder-structure-163069134`

### What are the relevant Pivotal Tracker stories?
#163069134

### Screenshots

![folderstr](https://user-images.githubusercontent.com/11965186/50898669-ea10f680-1420-11e9-8450-cb361eda25eb.png)
